test-automation-framework/org.wso2.carbon.automation.extensions/src/main/java/org/wso2/carbon/automation/extensions/servers/carbonserver/TestServerManager.java

```diff
@@ -80,6 +87,26 @@ public void configureServer() throws AutomationFrameworkException {
 
     }
 
+    /**
+     * Apply the configuration changes from the given confChangesLocation.
+     * If the confChangesLocation is empty will not do anything.
+     *
+     * @param confChangesLocation location where the desired config changes are.
+     * @throws AutomationFrameworkException throw if carbonHome is not defined.
+     * @throws IOException
```

Comment:
The Java doc generation in java8 fails if an argument/return/exception did not have a description.